docs/changelogs/CHANGELOG-2.26.md

```diff
@@ -0,0 +1,320 @@
+- Update configmap-reload to 0.12.0; container image is now pulled from `ghcr.io/jimmidyson/configmap-reload` instead of Docker Hub ([#13265](https://github.com/kubermatic/kubermatic/pull/13265))
+- Update flatcar-linux-update-operator to 0.9.0 ([#13666](https://github.com/kubermatic/kubermatic/pull/13666))
+- Update Helm version used by KKP to 3.14.3 ([#13244](https://github.com/kubermatic/kubermatic/pull/13244))
+- Update Karma to v1.120 ([#13277](https://github.com/kubermatic/kubermatic/pull/13277))
+- Update kube-dependencies to 0.29.3 ([#13186](https://github.com/kubermatic/kubermatic/pull/13186))
+- Update kube-state-metrics to v2.12 ([#13278](https://github.com/kubermatic/kubermatic/pull/13278))
+- Update KubeLB to v1.1.1 ([#13712](https://github.com/kubermatic/kubermatic/pull/13712))
+- Update node-exporter to v1.7.0 ([#13279](https://github.com/kubermatic/kubermatic/pull/13279))
+- Update Prometheus to v2.51.1 ([#13280](https://github.com/kubermatic/kubermatic/pull/13280))
+- Update to Go 1.23.1 ([#13711](https://github.com/kubermatic/kubermatic/pull/13711))
+- Update usercluster kube-state-metrics to 2.12.0 ([#13307](https://github.com/kubermatic/kubermatic/pull/13307))
+- Update Velero Helm chart to v1.13.1 ([#13272](https://github.com/kubermatic/kubermatic/pull/13272))
+- Update Velero to v1.14.0 ([#13473](https://github.com/kubermatic/kubermatic/pull/13473))
+
+### Cleanup
+
+- Add SecurityContext to KKP operator/controller-manager containers, including OSM and machine-controller ([#13282](https://github.com/kubermatic/kubermatic/pull/13282))
+- Addon conditions now contain the KKP version that has last successfully reconciled the addon (similar to the Cluster conditions) ([#13519](https://github.com/kubermatic/kubermatic/pull/13519))
+- Addons reconciliation is triggered more consistently for changes to Cluster objects, reducing the overall number of unnecessary addon reconciliations ([#13252](https://github.com/kubermatic/kubermatic/pull/13252))
+- Fix an issue which prohibited users to specify custom values for Cilium system application ([#13276](https://github.com/kubermatic/kubermatic/pull/13276))       
+- Fix misleading errors about undeploying the cluster-backup components from newly created user clusters ([#13403](https://github.com/kubermatic/kubermatic/pull/13403))
+- Replace custom Velero Helm chart with a wrapper around the official upstream chart ([#13488](https://github.com/kubermatic/kubermatic/pull/13488))
+- Replace kubernetes.io/ingress.class annotation with ingressClassName spec field ([#13549](https://github.com/kubermatic/kubermatic/pull/13549))
+- S3-Exporter does not run with root permissions and does not leak credentials via CLI flags anymore ([#13226](https://github.com/kubermatic/kubermatic/pull/13226))
+- Remove deprecated Cilium and Hubble KKP Addons, as Cilium CNI is managed by Applications ([#13229](https://github.com/kubermatic/kubermatic/pull/13229))
+
+
+### Deprecation
```

Comment:
Please go through the changelog and find _all_ the deprecations. If we have a dedicated section in the changelog for it, it should contain _all_ of them, not just the few PRs that were explicitly labelled as deprecations.

Reply:
Few are action required so left them as is. rest moved under deprecation section.